absearch/server.py

```diff
@@ -134,20 +137,29 @@ def handle_500_error(code):
 
 @app.route(PATH)
 def add_user_to_cohort(**kw):
-
-    with app._statsd.timer('add_user_to_cohort'):
+    if app._statsd:
+        with app._statsd.timer('add_user_to_cohort'):
+            res = app.settings.get(**kw)
```

Comment:
Please use timer.start and timer.stop instead as I did here: https://github.com/mozilla-services/syncto/pull/18/files#diff-3af1fdbac63877c891e6a68862dcf289R61